gcs/cloud-storage/template/template.yaml

```diff
@@ -0,0 +1,81 @@
+_p: &provider_gcp
+- google:
+    project: {{ .variables.project }}
+    region: {{ .variables.region }}
+
+name: gcs-static-website
+kind: StackTemplate
+units:
+  -
+    name: cloud-storage
+    type: tfmodule
+    providers: *provider_gcp
+    source: "github.com/terraform-google-modules/terraform-google-cloud-storage.git?ref=v4.0.1"
+    inputs:
+      names:
+        - {{ .variables.name }}
+      randomize_suffix: true
```

Comment:
remove if could be defaulted

Reply:
To avoid name collisions with existing buckets we should enable this option to randomize bucket name.